packages/analytics-js/src/components/userSessionManager/referrer.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Gets the URL of the location that referred the user to the current page.
+ * @returns string
+ */
+const getReferrer = () => document.referrer || '$direct';
```

Comment:
@sai has also done these as utils, take his once we merge

Reply:
Yes, I will remove this file once that PR is merged.

Reply:
The PR has been merged.

Reply:
Adressed